packages/mdctl-axon-tools/lib/StudyDataTranslations.js

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable camelcase */
+const globby = require('globby'),
+      path = require('path'),
+      fs = require('fs'),
+      _ = require('lodash'),
+      { privatesAccessor } = require('@medable/mdctl-core-utils/privates'),
+      { Driver } = require('@medable/mdctl-api-driver'),
+      { Org } = require('@medable/mdctl-api-driver/lib/cortex.object'),
+      { Fault } = require('@medable/mdctl-core')
+
+class StudyDataTranslations {
+
+  constructor(client, options = {}) {
+    Object.assign(privatesAccessor(this), {
+      client,
+      options
+    })
+  }
+
+  getOrg() {
+    const { client } = privatesAccessor(this),
+          driver = new Driver(client),
+          org = new Org(driver)
+
+    return org
+  }
+
+  isStudyDataTranslations({ input = process.cwd(), manifest }) {
```

Comment:
As this function deals with only authentication task, it might be a good idea to use specific name for this function
```suggestion
    isAuthTaskTranslations({ input = process.cwd(), manifest }) {
```